src/protocol/libp2p/kademlia/store.rs

```diff
@@ -660,4 +756,6 @@ mod tests {
         assert_eq!(store.get_providers(&provider2.key), vec![provider2]);
         assert_eq!(store.get_providers(&provider3.key), vec![]);
     }
+
+    // TODO: test local providers.
```

Comment:
nit: Maybe add an issue about this if not already?

Reply:
It's already addresses in the providers unit tests branch. I will remove this TODO in the upcoming PR ("part 6").